src/emailservice/email_server.py

```diff
@@ -143,7 +144,38 @@ def start(dummy_mode):
   except KeyboardInterrupt:
     server.stop(0)
 
+def initStackdriverProfiling():
+  enable_profiler = None
+  project_id = None
+  try:
+    enable_profiler = os.environ["ENABLE_PROFILER"]
+    project_id = os.environ["GCP_PROJECT_ID"]
+  except KeyError:
+    # Environment variable not set
+    pass
+  if enable_profiler != "1":
```

Comment:
I think it would be better if you do this check outside this method.

for example:


```
if os.environ('KEY', None) == '1':
   initStackdriverProfiler()
```

in the main code is better than this.

Reply:
Moved. Will upload a patch.